docs/getting-started/editor/Pulsar.md

```diff
@@ -0,0 +1,74 @@
+
+**Status**: Pulsar is still in Beta. New releases are available regularly. "The releases do seem to be working for most people and use cases." There is a known issue with MacOS performance. "Often times this can be resolved by disabling the `github` package."
+
+---
+**NOTE**: The projected timeline to have Pulsar ready with the Tidalcycles package is the end of 2022. Until then, new Tidal users are encouraged to wait for the new Pulsar editor or select one of the other editors. Tidal users who have a working Atom editor configured with the Tidalcycles package can continue to use it. But post Atom sunsetting, the Atom package manager will stop working.
+
+---
+
+## Manual Installation
+If you are comfortable with package installs - using git, adding npm, troubleshooting, etc.
+
+### Download & start Pulsar
+There are two ways to download:
+1. Pulsar [download page](https://pulsar-edit.dev/download.html). There are detailed instructions to get the latest stable build via Cirrus CI. Navigating Cirrus CI is a bit involved, so the 2nd method may be easier.
+2. Pulsar download link + add your specific query parameters:
+    - https://web.pulsar-edit.dev/download (need to add query string)
+    - https://github.com/pulsar-edit/package-frontend/blob/main/docs/download_links.md instructions on building your URL query string.
+    - For example: https://web.pulsar-edit.dev/download?os=intel_mac&type=mac_dmg
+
+When you start Pulsar the first time, it will create a hidden folder in your home directory `~/.pulsar`.
+
+### Manual install of Tidal package
+There is a Tidal community package to be manually installed: [atom-tidalcycles on github](https://github.com/tidalcycles/atom-tidalcycles)
+
+From a command line using git:
+
+```bash
+> cd ~/.pulsar/packages
+> git clone https://github.com/tidalcycles/atom-tidalcycles
+```
+
+Rename the new folder from `atom-tidalcycles` to `tidalcycles`. Without this, the Settings feature won't work properly.
+
+### Install the node.js modules
+If you have npm installed (Node package manager) you just need to run `npm install` from the tidalcycles directory: `~/.pulsar/packages/tidalcycles`
+
+```bash
+[userHome]/.pulsar/packages/tidalcycles > npm install
+```
+
+If you need npm, the recommended method is via nvm - Node Version Manager.
+- See [NPM Docs](https://docs.npmjs.com/downloading-and-installing-node-js-and-npm) instructions
+- Or go directly to the [nvm install script](https://github.com/nvm-sh/nvm) - has lots of detail, troubleshooting, support for different multiple OS, etc.
+- The nvm install script will add lines to your shell profile (.bash_profile, ~/.zshrc, ~/.bashrc etc). This can result in a short delay when starting the shell.
+- run `command -v nvm` to verify nvm install. (expected output `nvm`)
+- now use nvm to install npm:
+
+```bash
+> nvm install node
+```
+
+- Now the tidalcycles npm modules can be installed `npm install`. (see above)
```

Comment:
This whole how to install stanza should be moved above the `npm install` command referenced imo

Reply:
OK. I was thinking that perhaps most will already have npm installed, but that may not be the case. I can put the nvm details first - and add "If you already have npm, skip this step."